solvers/deepinv.py

```diff
@@ -0,0 +1,65 @@
+from benchopt import BaseSolver, safe_import_context
+
+with safe_import_context() as import_ctx:
+    import deepinv as dinv
+    import torch
+    from deepinv.optim.data_fidelity import L2
+
+
+class Solver(BaseSolver):
+    name = 'deepinv'
+
+    parameters = {
+        'recon': ['TV', 'DRUNet']
+    }
+
+    def skip(self, A, reg, delta, data_fit, y, isotropy):
+        if data_fit == 'huber':
+            return True, f"solver does not work with {data_fit} loss"
+        elif isotropy == 'anisotropic':
+            return True, f"solver does not work with {isotropy} regularization"
+        return False, None
+
+    def set_objective(self, A, reg, delta, data_fit, y, isotropy):
+        self.A, self.reg, self.y = A, reg, torch.from_numpy(y)
+        self.delta, self.data_fit = delta, data_fit
+        self.isotropy = isotropy
+
+    def run(self, n_iter):
```

Comment:
Use the [callback strategy](https://benchopt.github.io/user_guide/performance_curves.html#using-a-callback) here